subsys/shell/modules/kernel_service.c

```diff
@@ -218,6 +222,67 @@ static int cmd_kernel_stacks(const struct shell *shell,
 }
 #endif
 
+static int cmd_kernel_sleep(const struct shell *sh,
+			    size_t argc, char **argv)
+{
+	ARG_UNUSED(sh);
+	ARG_UNUSED(argc);
+
+	k_msleep(strtoul(argv[1], NULL, 10));
+
+	return 0;
+}
+
+#if defined(CONFIG_LOG_RUNTIME_FILTERING)
+static uint32_t log_source_id_get(const char *name)
+{
+	uint32_t count = log_src_cnt_get(CONFIG_LOG_DOMAIN_ID);
+
+	for (uint32_t i = 0; i < count; i++) {
+		if (strcmp(log_source_name_get(CONFIG_LOG_DOMAIN_ID, i), name)
+		    == 0) {
```

Comment:
This line split looks quite odd. Can't you fit it into a single line, assuming max line width of 100?